.github/workflows/master.yml

```diff
@@ -183,13 +183,18 @@ jobs:
         flink:
           - '1.16'
           - '1.17'
+          - '1.18'
         flink-archive: [ "" ]
         comment: [ "normal" ]
         include:
           - java: 8
             flink: '1.17'
             flink-archive: '-Dflink.archive.mirror=https://archive.apache.org/dist/flink/flink-1.16.1 -Dflink.archive.name=flink-1.16.1-bin-scala_2.12.tgz'
             comment: 'verify-on-flink-1.16-binary'
+          - java: 8
+            flink: '1.18'
+            flink-archive: '-Dflink.archive.mirror=https://archive.apache.org/dist/flink/flink-1.17.1 -Dflink.archive.name=flink-1.17.1-bin-scala_2.12.tgz'
+            comment: 'verify-on-flink-1.17-binary'
```

Comment:
Emm... this cross-verification is actually to sure the Flink engine which builds against the default Flink version can work well on all support Flink runtimes.

Specific to this case, it should be: using flink 1.17 for compilation, and verifying that it works well on Flink 1.18 runtime

Reply:
Thanks for your explanation. I've changed the flink version.